src/Processor/PaymentResponseProcessor/FallbackResponseProcessor.php

```diff
@@ -14,6 +14,9 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 
+/**
+ * @psalm-suppress PropertyNotSetInConstructor
```

Comment:
Why it's suppressed? If there is a reason, you've to describe it in a comment

Reply:
Removed psalm suppress, and added unused property to constructor in next commit.